NCI-GDC/gdc-frontend-framework
# PEAR-2248 - State migration for removed properties

## Description
To test:
1) Add some vestigial and non-vestigial custom filters on `develop`
2) Check out this branch and refresh
3) See that the vestigial properties have been removed from your custom filters

## Checklist

      flagged issues

## Screenshots/Screen Recordings (if Appropriate)
